frontend/src/lib/components/AppEditorLink/AppEditorLink.js

```diff
@@ -23,12 +24,17 @@ export function AppEditorLink({ actionId, style, className, children }) {
             >
                 {children}
             </a>
-            {modalOpen && (
-                <ChooseURLModal
-                    actionId={actionId}
-                    dismissModal={() => setModalOpen(false)}
-                />
-            )}
+            <Modal
+                visible={modalOpen}
+                title={
+                    actionId
+                        ? 'Where do you want to edit this action?'
+                        : 'On which domain do you want to create an action?'
```

Comment:
NIT: action statement rather than question: "Choose the domain where you want to create/edit an action"

Reply:
Yep good point!